sdk/ml/azure-ai-ml/azure/ai/ml/operations/_evaluator_operations.py

```diff
@@ -125,15 +127,15 @@ def _raise_if_not_evaluator(self, properties: Optional[Dict[str, Any]], message:
             )
 
     @monitor_with_activity(ops_logger, "Evaluator.Get", ActivityType.PUBLICAPI)
-    def get(self, name: str, version: Optional[str] = None, label: Optional[str] = None) -> Model:
+    def get(self, *, name: str, version: Optional[str] = None, label: Optional[str] = None, **kwargs) -> Model:
```

Comment:
Anna mentioned "version and label should be keyword-only." so I think it doesn't include name

Reply:
Fixed